Go/Symbols/GoConstSymbolIndex.tmPreferences

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+  <key>scope</key>
+  <string>
+    variable.other.constant.declaration.go - meta.block.go
```

Comment:
`entity.name.constant.go` would be more appropriate, assuming the syntax definition is updated as well.

Reply:
Agreed. The problem is that applying different const/var scopes depending on whether we're inside a block seems to require _lots_ of duplication in the syntax; not sure it's worth it.